code/datums/ammo/bullet/rifle.dm

```diff
@@ -174,7 +174,7 @@
 	shell_speed = AMMO_SPEED_TIER_6
 
 /datum/ammo/bullet/rifle/m4ra/impact/on_hit_mob(mob/M, obj/projectile/P)
-	knockback(M, P, 32) // Can knockback basically at max range max range is 24 tiles...
+	knockback(M, P, 12) //lowerd range to with in 2x scope
 
 /datum/ammo/bullet/rifle/m4ra/impact/knockback_effects(mob/living/living_mob, obj/projectile/fired_projectile)
```

Comment:
See this datum, it's what actually controls the stunning aspect of the rounds and if I remember correctly it does not or barely stuns past the default visible 7 tiles

Basically you need to be on screen

Reply:
Clarifying that its everything under /datum/ammo/bullet/rifle/m4ra/impact/knockback_effects(mob/living/living_mob, obj/projectile/fired_projectile)

Reply:
hmmm i did some testing before hand... but i am going to check again.

Reply:
After checking in my test i can confirm that the Rounds stun with no noticeable change in stun time at ranges longer than 7 seem to be the same. also my test confirm that that the range is beyond 21 tile range. this is for the master branch mind you. 

going to do some test with 
/datum/ammo/bullet/rifle/m4ra/impact/knockback_effects(mob/living/living_mob, obj/projectile/fired_projectile)